client/asset/btc/btc.go

```diff
@@ -431,21 +454,41 @@ func BTCCloneWallet(cfg *BTCCloneCFG) (*ExchangeWallet, error) {
 		Pass:         btcCfg.RPCPass,
 	}, nil)
 	if err != nil {
-		return nil, fmt.Errorf("error creating BTC RPC client: %w", err)
+		return nil, fmt.Errorf("error creating BTC RPC client: %v", err)
+	}
+
+	btc, err := newWallet(cfg, btcCfg, client)
+	if err != nil {
+		return nil, fmt.Errorf("error creating BTC ExchangeWallet: %v", err)
 	}
 
-	return newWallet(cfg, btcCfg, client), nil
+	return btc, nil
 }
 
 // newWallet creates the ExchangeWallet and starts the block monitor.
-func newWallet(cfg *BTCCloneCFG, btcCfg *dexbtc.Config, node rpcClient) *ExchangeWallet {
+func newWallet(cfg *BTCCloneCFG, btcCfg *dexbtc.Config, node rpcClient) (*ExchangeWallet, error) {
```

Comment:
Instead of adding an `error` return value here, can you just check the config in `BTCCloneWallet`, which already has the `error`?

Reply:
That's a good idea too. Although doing it in the caller, you'd have to keep the arguably redundant and error prone math of doing the check before the `toSatoshi( ... /1000)` that would definitively tell you if you actually hit the error condition.  I won't nit pick much more though.  The PR is very close to done and we can follow up.  Just a little cleanup as feels right for now is good with me.